### PR TITLE
Adds member data

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds MemberList data to the teams page. A hellip character has been added to the Actions column.

A separate story has been created to add a clickable menu item once a user clicks on the hellip and to make it so that it may only be seen by certain users, so that will be handled in a separate PR.

Fixes SC-15398

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Screenshot 2023-04-05 at 4 45 02 PM

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/teams/components/TeamsTable.tsx [hunk 1]
```diff
@@ -10,14 +33,43 @@ function TeamsTable() {
...
+          {
+            Header: 'Last Activity',
+            accessor: (date: any) => {
+              return formatDate(new Date(date.last_activity));
+            },
+          },
+          {
+            Header: 'Joined Date',
+            accessor: (date: any) => {
+              return formatDate(new Date(date.date_added));
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/teams/components/TeamsTable.tsx:57]: When I placed `formatDate` in the `util.ts` file the date didn't change.

File: web/beacon-app/src/features/teams/components/TeamsTable.tsx [hunk 2]
```diff
@@ -10,14 +33,43 @@ function TeamsTable() {
...
+            },
+          },
+          {
+            // TODO: Make actions viewable only to members with owner and admin permission
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/teams/components/TeamsTable.tsx:61]: This will be included in a separate PR.

File: web/beacon-app/src/utils/formatDate.ts [hunk 3]
```diff
@@ -1,14 +1,27 @@
 export const isDate = (value: any) => value instanceof Date;
 
-export const formatDate = (date: Date, format?: string) => {
+export const formatDate = (date: Date, format?: string, type?: string) => {
   if (!format) {
```
Comment (daniellemaxwell) [inline web/beacon-app/src/utils/formatDate.ts:4]: After making the change, the date format didn't change. They're returned as strings, so that's possibly why. Removing `hour` and `minute` here changed the date's format. However, I kept the addition that was suggested, so please let me know how you'd like to proceed.

**Review by masskoder — APPROVED** [review]
Looks good
